apps/web/src/app/tournament/[id]/create/_components/tournamentItemBasket/TournamentBasketItem.tsx

```
@@ -0,0 +1,17 @@
+import ProductImage from '@/components/common/product-image';
+import type { TournamentItemT } from '@/types/tournament';
+
+type TournamentBasketItemProps = {
+  item: TournamentItemT;
+  index: number;
+};
+
+function TournamentBasketItem({ item, index }: TournamentBasketItemProps) {
+  return (
+    <div className="relative aspect-square">
+      <ProductImage src={item.imageUrl ?? undefined} size="sm" fill alt={`토너먼트 아이템 ${index + 1}`} />
```

ESLint

[error] Unexpected use of undefined.

(no-undefined)